src/component/landing/index.js

```diff
@@ -1,11 +1,46 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button';
 import styles from './styles.module.css';
 
+function LeftSide() {
+  return (
+    <div className={styles.leftside}>
+      <h2>GT Web Dev's</h2>
+      <h1>Web Arcade</h1>
+      <p className={styles.bodyText}>Enter the Georgia Tech Web Dev Web Arcade to play
+         and rate user-made WebGL games or upload your own!</p>
+    </div>
+  );
+}
+
 function LandingPage() {
+  // store input text into variables
+  const [username, setUsername] = React.useState();
+  const [password, setPassword] = React.useState();
+  // event?.target?.value store the value of the input field
+  // value={username} the text in the username box (not the actual username)
+
+  // we call in the  onClick fuction in button
+  function signIn() {
+    // eslint-disable-next-line no-alert
```

Comment:
Thank you for your work on this! It seems like the style checker is not passing since ``// eslint-disable-next-line no-alert`` should be placed directly above ``alert('Not implemented');``. Once that is fixed, I think this should be good to merge!